docs/dg/dev/guidelines/performance-guidelines/general-performance-guidelines.md

```diff
@@ -37,6 +37,12 @@ Xdebug slows down the application and is not necessary for a non-development env
 
 For performance reasons, always use the newest stable version of PHP, as every new version ships with several improvements.
 
+## Packages with a performance fixes
+
+### Merchant portal related performance fixes:
```

Comment:
Shouldn't we also mention config changes https://github.com/spryker-shop/b2c-demo-marketplace/pull/527/files somewhere?

Reply:
Already published